main/integration-tests/__test__/api-tests/authentication/get-auth-provider-configs.test.js

```diff
@@ -0,0 +1,182 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('Get authentication config list scenarios', () => {
+  let setup;
+  let adminSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('Get authentication config list', () => {
+    it('should fail if user is not an admin', async () => {
+      const researcherSession = await setup.createResearcherSession();
+
+      await expect(researcherSession.resources.authentication.configs().get()).rejects.toMatchObject({
+        code: errorCode.http.code.forbidden,
+      });
+    });
+
+    it('should fail if user is inactive', async () => {
+      const admin2Session = await setup.createAdminSession();
+      await adminSession.resources.users.deactivateUser(admin2Session.user);
+
+      await expect(admin2Session.resources.authentication.configs().get()).rejects.toMatchObject({
+        code: errorCode.http.code.unauthorized,
+      });
+    });
+
+    it('should list auth provider configs', async () => {
+      const admin2Session = await setup.createAdminSession();
+      const response = await admin2Session.resources.authentication.configs().get();
+
+      await expect(response).toEqual(
+        expect.arrayContaining([
+          expect.objectContaining({
```

Comment:
If the output shape is 100% to match, can you use expect.toStrictEqual()?

Reply:
If you put this part in a file say 'expected-response.js' and then require it in the test, it help readability.  This is just a thought.

Reply:
The output also contains dateTime elements, due to which objectContaining seemed to fit. However `toEqual` can be safely replaced with `toStrictEqual`.

Reply:
Not changing this, per our conversation.